### DISCUSSION TITLE
Upscale Resolution?

### DISCUSSION THREAD

**Discussion by odvigajlo** [niels747/2D-Weather-Sandbox#13] [Ideas]
Maybe you could change size of each cell, so we could make more detailed simulations?

**Comment by Westerstaad** [comment]
would need a slider to control the resolution so your computer doesnt explode every time you load in

**Comment by niels747** [comment]
It wass possible before to change the vertical resolution, but precipitation and other things would behave very differently, that's why I took it out. I do eventually want it to behave the same at any vertical resolution. This requires compensation of all scale related parameters and it's a lot of work.

**Comment by niels747** [comment]
> Maybe you could change size of each cell, so we could make more detailed simulations?

So you want to run a vertical resolution higher than 300? You would need a really powerful gpu to run it at playable speed.
